src/robot/running/keywordrunner.py

```diff
@@ -17,7 +17,8 @@
                           HandlerExecutionFailed)
 from robot.result.keyword import Keyword as KeywordResult
 from robot.utils import (format_assign_message, frange, get_error_message,
-                         get_timestamp, plural_or_not as s, type_name)
+                         get_timestamp, plural_or_not as s, type_name,
+                         is_list_like, type_name)
```

Comment:
Importing `type_name` twice. Also, `is_list_like` should be placed after `get_timestamp` to keep imports in alphabetical order.